client_broadcast.go

```diff
@@ -74,18 +23,24 @@ func (c *Client) asyncBroadcastMsg(msgs ...sdk.Msg) (*txtypes.BroadcastTxRespons
 
 	res, err := c.broadcastTx(ctx, c.txFactory, msgs...)
 	if err != nil {
-		for range 5 {
+		for i := range 5 {
 			if err == nil {
 				break
 			}
 
 			if strings.Contains(err.Error(), "account sequence mismatch") {
-				c.syncNonce()
+				err = c.syncNonce()
+				if err != nil {
+					return nil, fmt.Errorf("syncNonce: %s", err)
```

Comment:
hmm, you might get 429 error or service temporary unavailable. Maybe we'll try again?? Just `continue`

Reply:
continue with log.Warnf() ?

Reply:
yep

Reply:
done